automatic/selenium-chromium-edge-driver/update.ps1

```diff
@@ -38,12 +36,9 @@ function global:au_SearchReplace {
 function global:au_GetLatest {
   # We use the available package on CCR as the source of truth
   $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
-  $escapedVersion = [regex]::Escape($packageVersion)
```

Comment:
I'm not entirely sure the reason it's here but I'm assuming it's for a good reason. Is there a reason this was removed?